### ISSUE TITLE
Always keep the group header on the screen unless the group overflows the check menu

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#1642]
The group header should always be visible on the screen unless there are too many elements in the group to all fit on the screen. In that case, the group header should remain visible until the first check is reached that would flow off the bottom.
May want to incorporate with #1325

**Comment by benjore** [comment]
The current check must always show on the screen.
The group header should always be on the screen whenever possible.

**Comment by cckozie** [comment]
@I'm not positive that this problem is related to keeping the group name on the screen, but this is what I found. Using this project https://door43.org/u/tc01/id_eph_text_ulb, I opened it with tN and did a bunch of checks and skips to test the group header staying on the screen (I had reduced the vertical height of the tC window to its minimum size). I then changed to tW with the same project, went to the checks for God, and they clicked the Save & Continue, then Skip, until the check scrolled to the bottom. After clicking Save & Continue on the last check on the screen, the app froze. I repeated the process just starting with tW and had the same result.

![image](https://user-images.githubusercontent.com/8171759/27405411-09c40178-5697-11e7-988d-ea183ffc015a.png)

**Comment by RoyalSix** [comment]
- [1] Fix Next Check Action Logic On Last Check

**Comment by cckozie** [comment]
beta.54
I'm still seeing a similar issue. I opened an Eph project with tW, clicked on the God group, then clicked Save & Continue and Skip on each check as they came up. When I did that on the check for Eph 2:6, the Skip/Close dialog did not disappear and the checking screen remained dim after I clicked the Skip button. It now continues to do that on each check.
![image](https://user-images.githubusercontent.com/8171759/28072143-baa45526-6617-11e7-9066-bc5c9c9908e8.png)
